lib/validate.js

```diff
@@ -83,12 +83,12 @@ export function advPostSchemaMembers (client) {
       .compact((v) => !v.nym && !v.pct)
       .test({
         name: 'sum',
-        test: forwards => forwards.map(fwd => Number(fwd.pct)).reduce((sum, cur) => sum + cur, 0) <= 100,
+        test: forwards => forwards ? forwards.map(fwd => Number(fwd.pct)).reduce((sum, cur) => sum + cur, 0) <= 100 : true,
```

Comment:
I'm surprised this gets invoked if it's not supplied?

Reply:
Good point, it's called even though there is no `.required(msg)` like for other attributes